### PR TITLE
[DA-1298] Hierarchy content implementation

### PR DESCRIPTION
**Pull request by wangy70**
Hierarchy content implementation

### CODE REVIEW

File: rest-api/dao/organization_hierarchy_sync_dao.py [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+    obj_type = self._get_type(hierarchy_org_obj)
+
+    operation_funcs = {
+      'AWARDEE': self._update_awardee,
+      'ORGANIZATION': self._update_organization,
+      'SITE': self._update_site
+    }
+
+    if obj_type not in operation_funcs:
+      raise BadRequest('No awardee-type info found in payload data.')
```
Comment (Seeker1911) [inline rest-api/dao/organization_hierarchy_sync_dao.py]: I think this wording could become confusing, maybe something without explicitly mentioning one of (awardee, organization, site)

File: rest-api/dao/organization_hierarchy_sync_dao.py [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+from model.organization import Organization
+from model.site import Site
+from model.site_enums import ObsoleteStatus, SiteStatus, EnrollingStatus, DigitalSchedulingStatus
+from dao.hpo_dao import HPODao
+from participant_enums import OrganizationType
+from dao.organization_dao import OrganizationDao
+from dao.site_dao import SiteDao
+from dateutil.parser import parse
+
+_FHIR_SYSTEM_PREFIX = 'http://all-of-us.org/fhir/sites/'
```
Comment (Seeker1911) [inline rest-api/dao/organization_hierarchy_sync_dao.py]: We have a similar structure in api_util.py. We might consider consolidating this constant there.

Comment (wangy70) [reply rest-api/dao/organization_hierarchy_sync_dao.py]: done

File: rest-api/dao/organization_hierarchy_sync_dao.py (hunk 1 above)
Comment (wangy70) [reply rest-api/dao/organization_hierarchy_sync_dao.py]: 'awardee-type' is the field name in their payload.

**Pull request merged**